boards/samr21-xpro/dist/gdb.conf

```diff
@@ -0,0 +1,3 @@
+target remote localhost:3333
+monitor reset halt
+load
```

Comment:
No debug symbols are stored on the device. `load` loads the program binary onto the target (i.e. flashing the device).
`file` loads debug symbols etc into GDB from the file system, but it is not necessary to give that command if you already gave the filename as a command line argument when starting GDB.